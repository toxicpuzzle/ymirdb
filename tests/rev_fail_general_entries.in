SET a 1 2 3 4 5 
SET b 1 2 3
PUSH b a
REV b
LIST ENTRIES
bye