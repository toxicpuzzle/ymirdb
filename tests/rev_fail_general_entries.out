> ok

> ok

> ok

> simple entry only

> b [a 1 2 3]
a [1 2 3 4 5]

> bye